Explorer/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 6000.2.6f2
```

Comment:
We can't revert this change, this is an important security patch. If this causes crash, we need to create unity support ticket and prioritize this with Unity directly